addons/l10n_zm/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
`# -*- coding: utf-8 -*-`
Are useless in all versions now, let's not add them in new files :slightly_smiling_face:

Reply:
Is it also the case for xml files ?
`<?xml version="1.0" encoding="utf-8"?>`

Reply:
No, not that I know. Only in python files.